configs/projects/bolt-runtime.rb

```diff
@@ -230,7 +230,7 @@
 
   # Components from puppet-runtime included to support apply on localhost
   # Only bundle SELinux gem for RHEL,Centos,Fedora
-  if platform.is_el? || platform.is_fedora?
+  if platform.is_el? || platform.is_fedora? || platform.is_debian?
```

Comment:
Will this break bolt on Debian 10 - 12 or Ubuntu < 24.04?

Reply:
It shouldn't break as puppet-runtime includes selinux gem support for ubuntu/debian,, but it should be a separate PR.
Removed bolt related changes.